ui/app/src/components/PublishDialog/PublishDialogContainer.tsx

```diff
@@ -83,68 +84,72 @@ export function PublishDialogContainer(props: PublishDialogContainerProps) {
       publishingTarget: null
     };
 
-    let itemsChecked = items.filter((item) => selectedItems[item.path]);
+    if (detailedItems) {
+      const itemsChecked = Object.keys(selectedItems)
+        .filter((path) => selectedItems[path])
+        .map((itemPath) => detailedItems[itemPath]);
```

Comment:
```suggestion
      const itemsChecked = Object.keys(selectedItems)
        .flatMap((itemPath) => selectedItems[itemPath] ? [detailedItems[itemPath]] : []);
```

Reply:
Per conversation, rolled back the change to lookup table of detailed items and a map to just use a flatMap instead of the filter of selectedItems